app/js/components/queryBuilder/queryParamSelector/queryParamSelector.tpl.html

```diff
@@ -42,7 +42,8 @@ <h2>{{$ctrl.currentState()}} Selector</h2>
 		is-searching="$ctrl.isSearching"
 		is-filter-disabled="$ctrl.currentState() === 'mutations'"
 	></cg-table>
-	<p ng-if="!$ctrl.searchResults.length && $ctrl.currentState() === 'mutations' && !$ctrl.isSearching">Begin searching to see results</p>
+	<p ng-if="!$ctrl.searchQuery.length && $ctrl.currentState() === 'mutations' && !$ctrl.isSearching">Begin searching to see results.</p>
+	<p ng-if="$ctrl.searchQuery.length && !$ctrl.searchResults.length && $ctrl.currentState() === 'mutations' && !$ctrl.isSearching">No results found.</p>
```

Comment:
These three conditions will have 8 states totally, two of which are handled here by `ng-if`. What about the other 6 states? We assume that they are all digested somewhere by the code?

Reply:
Which six states specifically?

Reply:
Here is what I meant: The code includes three boolean values:
(1) $ctrl.searchQuery.length === 0
(2) $ctrl.currentState() === 'mutations' 
(3) $ctrl.isSearching
Each boolean has two states (true or false), so the combination of these three will have 2x2x2 = 8 states totally. The two `ng-if` are handling 2 of them. What about the other 6 states?

Reply:
Yes, the other states are being digested by the code. 

When the current state is diseases, it automatically loads all the results and doesn't need to worry about the length of the search query (the user doesn't need to be prompted to start searching). And the `cgTable` component also has an `ng-if` on the "isSearching" boolean and only appears after that condition is `false`.